cfme/tests/openstack/cloud/test_instances.py

```diff
@@ -34,7 +34,8 @@ def new_instance(provider):
                     'last_name': fauxfactory.gen_alpha()},
         'catalog': {'num_vms': '1',
                     'vm_name': fauxfactory.gen_alpha()},
-        'environment': {'cloud_network': prov_data['cloud_network']},
+        'environment': {'cloud_network': prov_data['cloud_network'],
+                        'cloud_tenant': prov_data['cloud_tenant']},
```

Comment:
**QUESTION**
Does it work in CFME5.9?

Reply:
Yes, I executed regression tests for CFME 5.9.4.4 and test_create_instance works ok.